### ISSUE TITLE
Write some tests for the toy environments

### ISSUE THREAD

**Issue by david-lindner** [david-lindner/safe-grid-gym#12]
Tests are great!

**Comment by jvmncs** [comment]
Looks like there were some unit tests merged in #14. Seems like a good start, but we'll also want to write some integration tests for safe-grid-agents and this one. Mainly, we'll want to test that changes here don't introduce breaking changes with what we do over there. Could be as simple as a single training/evaluation iteration of Q learning on each of the environments

**Comment by david-lindner** [comment]
I think the additions from #17 are fine as unittests for the toy environments.

Integration tests between the repos make a lot of sense! Just to clarify how you imagine this: We would add tests to this repo, that import `safe-grid-agents`, run a training and evaluation iteration and check that nothing crashes and something reasonable happens, correct?

**Comment by jvmncs** [comment]
Precisely! That seems sufficient for now, at least until we have better unit testing over there.
